Announcements at release pages

grahamperrin opened the pull request:
At each index page, list the announcement.

Whilst here: 

- consistent use of trailing '/' for links to the listed pages
- consistent use of commas
- one sentence per line.

cperciva commented:
Thanks for this @grahamperrin.  A few questions/requests:

* Why do the links need trailing `/` characters?

* Can you add the missing "announce" link to `website/archetypes/release` as well so it will show up automatically in future releases?

* Can you clean up the git history (fix the commit message typo and get rid of the merge commit)?

grahamperrin commented:
> * Why do the links need trailing `/` characters?

Whilst not strictly necessary, the solidus avoids the [301 Moved Permanently](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/301) redirection response. 

With one of the links from current <https://www.freebsd.org/releases/14.1R/> as an example: 

![image](https://github.com/user-attachments/assets/7c89acf6-d438-4915-a55a-8225236c43ad)

grahamperrin commented:
> * Can you clean up the git history (fix the commit message typo and get rid of the merge commit)?

Note to self: [Amending older or multiple commit messages](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message#amending-older-or-multiple-commit-messages), `git rebase -i HEAD~n` etc.

cperciva commented:
> I can also do 10 (from 10.2), 11, and 12.
> 
> Yes or no?

Not worth bothering with ancient releases, IMHO.  Probably the only thing looking at those release notes now is AI scrapers.

grahamperrin commented:
Re: <https://github.com/freebsd/freebsd-doc/pull/508#issuecomment-2974258889> above, would you like to use the `.diff` there? Commit as "Reported by Graham Perrin" or something, I'm not fussed. 

----

Much of my Git knowledge has vaporised … the first of the commits above was in mid-March, I can't figure out how to remove the subsequent merge commit, et cetera, and I _really_ don't want to waste anyone's time …

<details>
<summary>Details</summary>

```text
grahamperrin@mowa219-gjp4-zbook-freebsd /u/doc (grahamperrin-releases)> git rebase -i 81cb0456a821f7e7ab67ac9201fa39dbab228a6d^
Auto-merging website/themes/beastie/layouts/partials/sidenav.html
CONFLICT (content): Merge conflict in website/themes/beastie/layouts/partials/sidenav.html
error: could not apply 54bab98bc2... sidenav.html three releases are legacy
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
hint: Disable this message with "git config set advice.mergeConflict false"
Could not apply 54bab98bc2... sidenav.html three releases are legacy
grahamperrin@mowa219-gjp4-zbook-freebsd /u/doc (grahamperrin-releases|REBASE-i 1/13) [1]> git rebase -i 81cb0456a821f7e7ab67ac9201fa39dbab228a6d^
fatal: It seems that there is already a rebase-merge directory, and
I wonder if you are in the middle of another rebase.  If that is the
case, please try
        git rebase (--continue | --abort | --skip)
If that is not the case, please
        rm -fr ".git/rebase-merge"
and run me again.  I am stopping in case you still have something
valuable there.

grahamperrin@mowa219-gjp4-zbook-freebsd /u/doc (grahamperrin-releases|REBASE-i 1/13) [128]> rm -fr ".git/rebase-merge"
grahamperrin@mowa219-gjp4-zbook-freebsd /u/doc ((23765fc2))> git reset --hard HEAD
HEAD is now at 23765fc27a 13.4R: announcement
grahamperrin@mowa219-gjp4-zbook-freebsd /u/doc ((23765fc2))> git rebase -i 81cb0456a821f7e7ab67ac9201fa39dbab228a6d^
Successfully rebased and updated detached HEAD.
grahamperrin@mowa219-gjp4-zbook-freebsd /u/doc ((23765fc2))> git switch freebsd main
fatal: only one reference expected
grahamperrin@mowa219-gjp4-zbook-freebsd /u/doc ((23765fc2)) [128]> git switch main
Previous HEAD position was 23765fc27a 13.4R: announcement
Switched to branch 'main'
Your branch is ahead of 'origin/main' by 700 commits.
  (use "git push" to publish your local commits)
grahamperrin@mowa219-gjp4-zbook-freebsd /u/doc (main)> git pull --ff-only
Already up to date.
grahamperrin@mowa219-gjp4-zbook-freebsd /u/doc (main)> git branch
  2023q3_ff
  events
  grahamperrin-14-schedule
  grahamperrin-bug-255318
  grahamperrin-bug-273918
  grahamperrin-bug-274072
  grahamperrin-faq
  grahamperrin-get
  grahamperrin-patch-notag
  grahamperrin-releases
  grahamperrin-status-portmgr
  grahamperrin-status-process
  grahamperrin-wifibox
* main
  main-grahamperrin
  outpaddling-main
  patch-13
  publish
grahamperrin@mowa219-gjp4-zbook-freebsd /u/doc (main)> 
```

… et cetera, I hope I'm back on `freebsd main` (no longer in a mess with my `grahamperrin-releases` branch, although my local `/usr/doc/website/archetypes/release` was apparently modified today (unlike the date of <https://github.com/freebsd/freebsd-doc/blob/main/website/archetypes/release/_index.adoc>) and does not have the _announce_ line. 

```text
grahamperrin@mowa219-gjp4-zbook-freebsd /u/d/w/a/release (main)> ls -hlrt
total 36 KB
-rw-r--r--  1 grahamperrin wheel  2.2K 20 Mar  2024 errata.adoc
-rw-r--r--  1 grahamperrin wheel  9.6K 20 Mar  2024 readme.adoc
-rw-r--r--  1 grahamperrin wheel  6.8K 14 Oct  2024 relnotes.adoc
-rw-r--r--  1 grahamperrin wheel  3.0K 14 Oct  2024 schedule.adoc
-rw-r--r--  1 grahamperrin wheel  2.8K 11 Dec  2024 installation.adoc
-rw-r--r--  1 grahamperrin wheel  998B 21 Jun 09:34 _index.adoc
-rw-r--r--  1 grahamperrin wheel   16K 21 Jun 09:41 hardware.adoc
grahamperrin@mowa219-gjp4-zbook-freebsd /u/d/w/a/release (main)> grep announce _index.adoc 
grahamperrin@mowa219-gjp4-zbook-freebsd /u/d/w/a/release (main) [1]> pwd
/usr/doc/website/archetypes/release
grahamperrin@mowa219-gjp4-zbook-freebsd /u/d/w/a/release (main)> cd
grahamperrin@mowa219-gjp4-zbook-freebsd ~> 
```

</details>

cperciva commented:
Thanks for all of this.  I reworked the changes to have one commit per "type" of change, and took the opportunity to get rid of broken "todo" links.  I *think* I got all of your changes... let me know if you notice that I missed anything!

grahamperrin commented:
Oh, thanks! 

I didn't intend for _you_ to rework all of that (I could have begun afresh with a new branch and a new PR). 

----

Links, for convenience (including two that do not yet exist): 

* https://www.freebsd.org/releases/10.2R/
* https://www.freebsd.org/releases/10.3R/
* https://www.freebsd.org/releases/10.4R/
* https://www.freebsd.org/releases/11.0R/
* https://www.freebsd.org/releases/11.1R/
* https://www.freebsd.org/releases/11.2R/
* https://www.freebsd.org/releases/11.3R/
* https://www.freebsd.org/releases/11.4R/
* https://www.freebsd.org/releases/12.0R/
* https://www.freebsd.org/releases/12.1R/
* https://www.freebsd.org/releases/12.2R/
* https://www.freebsd.org/releases/12.3R/
* https://www.freebsd.org/releases/12.4R/
* https://www.freebsd.org/releases/13.0R/
* https://www.freebsd.org/releases/13.1R/
* https://www.freebsd.org/releases/13.2R/
* https://www.freebsd.org/releases/13.3R/
* https://www.freebsd.org/releases/13.4R/
* https://www.freebsd.org/releases/13.5R/
* https://www.freebsd.org/releases/14.0R/
* https://www.freebsd.org/releases/14.1R/
* https://www.freebsd.org/releases/14.2R/
* https://www.freebsd.org/releases/14.3R/
* https://www.freebsd.org/releases/14.4R/
* https://www.freebsd.org/releases/14.5R/
* https://www.freebsd.org/releases/15.0R/

cperciva commented:
I ask you to do enough proofreading for me, it would be churlish of me to complain when you proofread things and find mistakes. ;-)
